airbnb_project/database/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# Use the official Docker Hub Postgres image
+FROM postgres
+
+# Set the working directory
+WORKDIR /var/lib/postgresql/data
```

Comment:
why you selected this directory?

Reply:
So this is for the case where we wanna compose the "app", say on a EC2 instance, that will be the default directory to which the app will be installed in. This could be anything. Matter of fact, this could simply be `/app` but I'm assuming it goes deeper in hierarchy because it's a db and not an application.